src/components/Courses/CourseInfo/CourseMeta.js

```diff
@@ -5,9 +5,16 @@ import { FaRegClock, FaRegCalendarAlt, FaInfoCircle } from 'react-icons/fa'
 import { DateTime } from 'luxon'
 import { calculateVideosDuration } from '../../../common/util'
 import Badge from '../../Badge'
+import useLanguage from '../../../hooks/useLanguage'
 
-const CourseMeta = ({ lectures, createdAt, tags }) => {
+const CourseMeta = ({ lectures, tags }) => {
   const { t } = useTranslation()
+  const { language } = useLanguage()
+  const courseCreatedDate = DateTime.fromISO(lectures[0].created_at)
+  const courseDate =
+    language === 'ar'
```

Comment:
@alaa-yahia changed it to get the date from the first lecture, and also to localise the formatted date in Arabic if the language is Arabic (otherwise it uses the computer default language)